src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+export const UserInfo = ({ user }) => {
+  const { name, email } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto: ${email}`}>
```

Comment:
Fix the href please 
https://css-tricks.com/snippets/html/mailto-links/
![image](https://github.com/mate-academy/react_static-list-of-todos-js/assets/99403198/b0e59d0b-6785-4034-a351-8fedf0ab45d0)